packages/localizations/src/en-US.ts

```diff
@@ -548,6 +548,62 @@ export const enUS: LocalizationResource = {
     action__signOut: 'Sign out',
     action__signOutAll: 'Sign out of all accounts',
   },
+  __experimental_userVerification: {
+    alternativeMethods: {
+      actionLink: 'Get help',
+      actionText: 'Don’t have any of these?',
+      blockButton__backupCode: 'Use a backup code',
+      blockButton__emailCode: 'Email code to {{identifier}}',
+      blockButton__password: 'Continue with your password',
+      blockButton__phoneCode: 'Send SMS code to {{identifier}}',
+      blockButton__totp: 'Use your authenticator app',
+      getHelp: {
+        blockButton__emailSupport: 'Email support',
+        content:
+          'If you’re experiencing difficulty verifying your account, email us and we will work with you to restore access as soon as possible.',
```

Comment:
Reads better to me

```suggestion
          'If you have trouble verifying your account, email us and we will work with you to restore access as soon as possible.',
```

Reply:
No problem just letting you know that the above is copied from what we already have for sign-in. We should either noop, or update both.

Reply:
Then let's update both, "If you’re experiencing difficulty" is more complex than necessary

Reply:
no problem 😉